01-HelloWorld/hello.c

```diff
@@ -7,6 +7,8 @@
  */
 #define USART_FLAG_TXE	((uint16_t) 0x0080)
 
+char greet[] = "Hello World!\n";
```

Comment:
Mark it as "static". Explain the reason why straightforward string literals do not work.